download/goodies/sfml-icon.svg

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<svg width="130" height="130" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>SFML Logo</title>
+ <defs>
+  <linearGradient id="linearGradient4987">
+   <stop stop-color="#a9ee52" offset="0" id="stop4989"/>
+   <stop stop-color="#53880e" offset="1" id="stop4991"/>
+  </linearGradient>
+  <linearGradient xlink:href="#linearGradient4987" id="linearGradient4885" y2="1.72718" x2="0.681638" y1="-0.03212" x1="0.681638"/>
+  <linearGradient xlink:href="#linearGradient4987" id="linearGradient4923" y2="1.229777" x2="0.726254" y1="-0.801712" x1="0.726254"/>
+ </defs>
+ <metadata id="metadata7">image/svg+xmlSFML Logo1 May 2013Michael Bradshaw</metadata>
```

Comment:
Not a big deal, but maybe we should copy the meta data from [logo](https://github.com/SFML/SFML-Website/blob/logos/download/goodies/sfml-logo.svg?short_path=35ec49b#L95) since it includes a lot more information.